app-misc/ola: Fix compilation against media-libs/liblo-0.32

kripton opened the pull request:
Closes: https://bugs.gentoo.org/927000

Patch taken directly from upstream: https://github.com/OpenLightingProject/ola/pull/1954.patch

orbea commented:
Please tag your commit message with `Upstream-PR:` and `Upstream-Commit:`, you can look at the commit history for other examples.

Additionally its helpful to add URLs for the Gentoo issue and any relevant upstream URLS to the top of the patch file for future reference. As well a comment for the URL to the Gentoo issue in the ebuild itself above the patch.

kripton commented:
@orbea: Thanks for the review :+1: . I was not aware of the `Upstream-PR` and `Upstream-Commit` tags. I added them, considered your other comments and rebased

kripton commented:
@ulm : Could I politely ask you to have a look? This PR adds a patch from upstream's git repo that fixes compilation against a library in Gentoo. The old version has already been tree-cleaned. Thank you!
